### PR TITLE
Remove requirement for config file, check for station_id key

### PR DESCRIPTION
**Pull request by wallacbe**
Allow to launch openhtf tester from command line without a config file

### CODE REVIEW

File: openhtf/conf.py [hunk 1]
```diff
@@ -257,29 +256,31 @@ def Load(self, config_file=None, force_reload=False,
...
+            raise ConfigurationInvalidError('No data', config_file)
+          self._state.clear()
+          self._state.update(data)
 
       # Load string values from flags
       for keyval in FLAGS.config_value:
         key, val = keyval.split('=')
         self._state[key] = val
+   
+      if 'station_id' not in self._state:
```
Comment (fahhem) [inline openhtf/conf.py]: Is this necessary for openhtf to function now? Or only for the frontend or something?

Comment (wallacbe) [inline openhtf/conf.py]: If station_id key isn't present we see the following:

Traceback (most recent call last):
  File "./hello_world.py", line 91, in <module>
    test.Execute(test_start=triggers.PromptForTestStart())
  File "/usr/local/google/home/wallaceb/openhtf/openhtf/__init__.py", line 204, in Execute
    os.getpid()).SaveToFile(FLAGS.rundir)
  File "/usr/local//home/openhtf/openhtf/io/rundata.py", line 71, in SaveToFile
    filename = os.path.join(directory, self.station_id)
  File "/usr/local//home/venv/lib/python2.7/posixpath.py", line 75, in join
    if b.startswith('/'):
AttributeError: 'NoneType' object has no attribute 'startswith'

Comment (jettisonjoe) [inline openhtf/conf.py]: The rundata module uses it as we see, and the built-in JSON output module uses it as well.

**Comment by wallacbe** [comment]
Notes from meeting:
Add default option to conf Declare function
provide a default station ID

**Pull request closed without merging**